Source/Array.swift

```diff
@@ -8,7 +8,7 @@
 
     - returns: A value of type `[U]`
 */
-public func <^> <T, U>(f: T -> U, a: [T]) -> [U] {
+public func <^> <T, U>(@noescape f: T -> U, a: [T]) -> [U] {
```

Comment:
Variable Name Min Length Rule Violation: Variable name should be 2 characters or more: currently 1 characters (variable_name_min_length)